### PR TITLE
Return from runtime start.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: riscv/src/compiler.rs [hunk 1]
```diff
@@ -1252,10 +1253,6 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             // has been recognized.
             match args {
                 [label] => match label {
-                    // if we call `call main;`, we replace with `call main; return;`
-                    Argument::Expression(Expression::Symbol(l)) if l == "main" => {
-                        vec!["call main;".into(), "return;".into()]
-                    }
```
Comment (leonardoalt) [inline riscv/src/compiler.rs:1258]: I think we need to remove the `tail` one as well

**Comment by chriseth** [comment]
Also reduced code duplication.

**Comment by leonardoalt** [comment]
witgen is now failing for keccak and vec_median

**Comment by leonardoalt** [comment]
and `tail` still missing

**Comment by chriseth** [comment]
Removed more code duplication.

**Comment by chriseth** [comment]
Hopefully fixed now.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**